src/index.json

```diff
@@ -23619,6 +23619,73 @@
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "Programming Language :: Python :: 3.9",
+                        "Programming Language :: Python :: 3.10",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
+                                    "name": "Microsoft Corporation",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/azure-cli-extensions/tree/master/src/machinelearningservices"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "ml",
+                    "run_requires": [
+                        {
+                            "requires": [
+                                "applicationinsights (<=0.11.10)",
+                                "azure-common (<2.0.0,>=1.1)",
+                                "azure-identity",
+                                "azure-storage-blob (<=12.9.0,>12.0.0b4)",
+                                "azure-storage-file-share (<13.0.0)",
+                                "colorama (<=0.4.4)",
+                                "cryptography (<=3.3.2)",
+                                "docker",
+                                "isodate",
+                                "marshmallow (<4.0.0,>=3.5)",
+                                "pathspec (==0.9.*)",
+                                "pydash (<=4.9.0)",
+                                "pyjwt (<=2.3.0)",
+                                "tqdm (<=4.63.0)",
+                                "typing-extensions (<4.0.0,>=3.10)"
+                            ]
+                        }
+                    ],
+                    "summary": "Microsoft Azure Command-Line Tools AzureMachineLearningWorkspaces Extension",
+                    "version": "2.2.3"
+                },
+                "sha256Digest": "6CBFAC0985961D82AE22FE7BCCAC9CEE8A26C78958756F463CED58913EF96409"
```

Comment:
```suggestion
                "sha256Digest": "6cbfac0985961d82ae22fe7bccac9cee8a26c78958756f463ced58913ef96409"
```

Reply:
Already pushed the change for hash in lower case.